README.md

```diff
@@ -7,7 +7,8 @@ applications targeting all major HPC platforms. For that purpose it provides
 abstractions for both parallel execution of code and data management.
 Kokkos is designed to target complex node architectures with N-level memory
 hierarchies and multiple types of execution resources. It currently can use
-OpenMP, Pthreads and CUDA as backend programming models with several other backends in development.
+CUDA, HPX, OpenMP and Pthreads as backend programming models with several other
```

Comment:
Do we "officially" support HPX?

Reply:
If not, we should mark this as experimental or something the like in the description of the corresponding `CMake` variable.